lumicks/pylake/kymo.py

```diff
@@ -570,6 +572,74 @@ def plot_with_time_histogram(
         ax_hist.set_ylabel("counts")
         ax_hist.set_title(self.name)
 
+    def bead_roi(self, bead_diameter, *, channel="green", plot=False, percentile=70, **kwargs):
+        """Determine approximate positional coordinates of the bead edges.
+
+        Searches for the bead edges by checking where the fluorescence drops below a threshold.
+        Only intended for non-moving beads.
+
+        .. warning::
+
+            This is early access alpha functionality. While usable, this has not yet been tested in
+            a large number of different scenarios. The API can still be subject to change without
+            any prior deprecation notice! If you use this functionality keep a close eye on the
+            changelog for any changes that may affect your analysis.
+
+        Parameters
+        ----------
+        bead_diameter : float
+            Rough estimate for the bead size (microns).
+        channel : 'red', 'green', 'blue', optional
+            Channel to use for bead detection.
+        plot : bool, optional
+            Plot result
+        percentile : int
+            Percentile to drop down to before accepting that we have left the bead area. Higher
+            values will make the bounds go closer to the bead edge but risk the algorithm failing
+            or having high background.
+        **kwargs
+            Forwarded to bead finding function.
+
+        Raises
+        ------
+        RuntimeError
+            When the algorithm fails to locate two edges during the bead finding stage.
+        """
+
+        return (
+            find_beads(
+                self.get_image(channel).sum(axis=1),
+                bead_diameter_pixels=bead_diameter / self.pixelsize_um[0],
+                plot=plot,
+                percentile=percentile,
+                **kwargs,
+            )
+            * self.pixelsize_um[0]
+        )
+
+    def crop_beads(self, bead_diameter, channel="green", **kwargs):
+        """Crop beads out of kymograph
```

Comment:
maybe something like:

```suggestion
        """Estimates the edges of stationary beads and returns a copy of the kymograph cropped to these limits.
```